index.html

```diff
@@ -6,174 +6,186 @@
-        <img src="images/mobile-menu-image2.png" alt="Dotts">
-      </div>
-      <div class="menu_mobile">
-        <ul class="menu_items">
-          <li> <a href="#home">Home</a></li>
-          <li> <a href="#about">About</a></li>
-          <li> <a href="#portfolio">Portfolio</a></li>
-          <li> <a href="#vontact">Contact</a></li>
-        </ul>
-      </div>
+ <nav>
+   <div class="mob_ver_menubar">
+   <img class="logo" src="images/1.png" alt="Website_logo_mv">
+   <img class="toggle" id="hamburger" src="images/icons8-menu-.png" alt="Toggle">
+   </div>
+   <ul class="desktop-menu">
+     <li><a href="#home">Home</a></li>
+     <li><a href="#portfolio">Portfolio</a></li>
+     <li><a href="#about">About</a></li>
+     <li><a href="#contact">Contact</a></li>
+   </ul>
 
-      <div class="img-2">
-        <img src="images/mobile-menu-image2.png" alt="dotts">
-      </div>
-      
-      <div class="img-3">
-        <img src="images/Mask_Group.png" alt="Logo-end">
-      </div>
-      
-
-
-      <!-- <ul class="menu_items">
-        <li> <a href="#">Home</a></li>
-        <li> <a href="#">About</a></li>
-        <li> <a href="#">Portfolio</a></li>
-        <li> <a href="#">Contact</a></li>
-      </ul> -->
-    </div>
-  </nav>
+   <!-- display menu with JavaScript -->
+   <div class="menu">
+     <div class="closebtn">
+       <img id="exitbtn" src="images/mobile_menu_cross.png" alt="close">
+     </div>
+   <ul id="display_menu_Js">
+     <li><a href="#home">Home</a></li>
+     <li><a href="#portfolio">Portfolio</a></li>
+     <li><a href="#about">About</a></li>
+     <li><a href="#contact">Contact</a></li>
+   </ul>
+   </div>
+ </nav>
 </header>
 
-<!-- Self Introduction Section -->
-<section class="intro">
-  <section class="name_title">
-  <h1>Fatima Zahir</h1>
-  <p>I am a Software Engineer, Professional Web Developer & Social Media Manager. I have developed a number of websites. I am an expert in HTML, CSS, JS, Ruby on Rails, PHP, jquery, MySQL. I can develop and design any type of website and fix errors.</p>
-  <button type="button" class="intro-button">Start collaboration</button> 
-</section>
-
-  <section class="desktop_image">
-    <img src="images/content-img.png" alt="Content Image">
-  </section> 
-
+<!-- Introduction Section -->
+<section class="intro" id="home">
+ <div class="name_title">
+ <h1>Fatima Zahir</h1>
+ <p>I am a Software Engineer, Professional Web Developer & Social Media Manager. I have developed a number of websites. I am an expert in HTML, CSS, JS, Ruby on Rails, PHP, jquery, MySQL. I can develop and design any type of website and fix errors.</p>
+ <button type="button" class="intro-button">Start collaboration</button> 
+ </div>
+ <section class="desktop_image">
+   <img src="images/content-img.png" alt="Content Image">
+ </section> 
 </section>
-
 <img class="end-intro-section" src="images/2.png" alt="Design">
 
 <!-- Work Section -->
 <section class="work-section" id="portfolio">
```

Comment:
- [ ] For this project You need to update the main page so that the projects section is created dynamically using JavaScript. Remember, all of the HTML in that section is created when the page loads. Please make sure you update the project section and it should be created dynamically using Javascript.

### See the image of the requirement

![image](https://user-images.githubusercontent.com/83803414/163249173-4e2a2f1f-d877-4450-827d-a418215289ce.png)